src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -102,21 +100,28 @@ run_teuthology_tests() {
     find ../src/pybind/mgr/dashboard/ -name '*.pyc' -exec rm -f {} \;
     source $TEMP_DIR/venv/bin/activate
 
-
-    if [ "$#" -gt 0 ]; then
-      TEST_CASES=""
+    OPTIONS=''
+    TEST_CASES=''
+    if [[ "$@" == '' || "$@" == '--create-cluster-only' ]]; then
+      TEST_CASES=`for i in \`ls $BUILD_DIR/../qa/tasks/mgr/dashboard/test_*\`; do F=$(basename $i); M="${F%.*}"; echo -n " tasks.mgr.dashboard.$M"; done`
+      TEST_CASES="tasks.mgr.test_dashboard $TEST_CASES"
+      if [[ "$@" == '--create-cluster-only' ]]; then
+        OPTIONS="$@"
+      fi
+    else
       for t in "$@"; do
         TEST_CASES="$TEST_CASES $t"
       done
-    else
-      TEST_CASES=`for i in \`ls $BUILD_DIR/../qa/tasks/mgr/dashboard/test_*\`; do F=$(basename $i); M="${F%.*}"; echo -n " tasks.mgr.dashboard.$M"; done`
-      TEST_CASES="tasks.mgr.test_dashboard $TEST_CASES"
     fi
 
     export PATH=$BUILD_DIR/bin:$PATH
     export LD_LIBRARY_PATH=$BUILD_DIR/lib/cython_modules/lib.${CEPH_PY_VERSION_MAJOR}/:$BUILD_DIR/lib
     export PYTHONPATH=$TEMP_DIR/teuthology:$BUILD_DIR/../qa:$BUILD_DIR/lib/cython_modules/lib.${CEPH_PY_VERSION_MAJOR}/:$BUILD_DIR/../src/pybind
-    eval python ../qa/tasks/vstart_runner.py $TEST_CASES
+    if [[ -z "$RGW" ]]; then
+        export RGW=1
```

Comment:
This env variable is not visible when starting the vstart cluster from within the vstart_runner.py script. In my system, the RGW daemon is not created and therefore any test relying on RGW fails.

I think you have two possible approaches to solve this: either run vstart.sh from this bash script (as we currently do), or add the support for passing that `$RGW` variable into the vstart_runner.py script and make sure that vstart_runner.py script starts the cluster with the RGW daemons.